pkgs/by-name/pr/prettier-plugin-go-template/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage rec {
+  pname = "prettier-plugin-go-template";
```

Comment:
```suggestion
  pname = "prettier-plugin-go-template";
  version = "0-unstable-2023-07-26";
```

Reply:
The commit date of https://github.com/NiklasPor/prettier-plugin-go-template/commit/d91c82e1377b89592ea3365e7e5569688fbc7954.